lib/mongoose_push_web/schemas/request/send_notification/deep.ex

```diff
@@ -0,0 +1,35 @@
+defmodule MongoosePushWeb.Schemas.Request.SendNotification.Deep do
+  require OpenApiSpex
+  alias OpenApiSpex.Schema
+
+  OpenApiSpex.schema(%{
+    title: "Request.SendNotification.Deep",
+    description: "Push notification request basic schema",
+    type: :object,
+    properties: %{
+      service: %Schema{
+        type: :string,
+        description: "Push notification service",
+        format: :string,
+        enum: ["fcm", "apns"]
+      },
+      mode: %Schema{type: :string, enum: ["prod", "dev"]},
+      priority: %Schema{
+        type: :string,
+        description: "The default one is chosen based on the service being used",
+        enum: ["normal", "high"]
+      },
+      time_to_live: %Schema{type: :integer, format: :int32},
+      mutable_content: %Schema{type: :boolean, default: false},
+      tags: %Schema{
+        type: :array,
```

Comment:
This definition doesn't seem to match `array` data type defined here: https://swagger.io/docs/specification/data-models/data-types/
According to docs, items' type has to be provided, and `format` doesn't seem to be it.

Reply:
You're right and I've overseen this detail in docs. I tried it wrong way before and gave up finally.